bitcoin-sv/ts-sdk
# [feature] add random fee calc benford approx

## Description of Changes

Adding Benford approximate random change outputs mode.

## Testing Procedure

Manual testing only. We should add tests to see if it sets change properly for many outputs.

- [x] I have tested manually in my local environment

## Checklist:

- [x] I have run `npm run doc` and `npm run lint` one final time before requesting a review
- [x] I have run `npm version patch` so that my changes will trigger a new version to be released when they are merged
